docs/getting-started.mdx

```diff
@@ -33,31 +33,20 @@ Your project should now look like this:
 
 ### Running your app
 
-The Nitric CLI can run your application locally for development and testing.
+The Nitric CLI starts the Nitric Server using `nitric start`. This will listen for any requests that are made from your code. Once the server has started, you can run your functions like any regular node application. In the starter templates, we provide a recommended setup so you can start the server and your code using npm or yarn.
 
 ```bash
-nitric run
-```
-
-Output:
-
-```txt
- SUCCESS  Configuration gathered (2s)
- SUCCESS  Created Dev Image! (0s)
- SUCCESS  Started Local Services! (2s)
- SUCCESS  Started Functions! (0s)
-Local running, use ctrl-C to stop
-
-Api  | Endpoint
-main | http://localhost:9001/apis/main
+npm run dev
+OR
+yarn dev
```

Comment:
this is going to get a bit messy when we have different runtimes..

IMHO we should defer to the user instead of getting fancy with this..

Run `nitric start` in one terminal then `nodemon -r dotenv/config ./functions/*.js` in another.